client/core/core.go

```diff
@@ -2793,7 +2793,7 @@ func (c *Core) initialize() {
 			defer wg.Done()
 			host, err := addrHost(acct.Host)
 			if err != nil {
-				c.log.Errorf("skipping loading of %s due to address parse error: %w", acct.Host, err)
+				c.log.Errorf("skipping loading of %s due to address parse error: %v", acct.Host, err)
```

Comment:
%w really only applies to wrapping `error` values, and in fact some libraries (maybe our logger) can panic on unrecognized printf specifiers.